ClinGen/clincoded
# Connects to #1566. Help docs on GitHub Wiki.

**Steps to test (part 1):**
1. Without login, select the **Gene Curation** and/or **Variant Curation** options from the **Help** dropdown menu in the top nav.
2. Expect to be taken to a new window/tab showing either the https://github.com/ClinGen/clincoded/wiki/GCI-Curation-Help or https://github.com/ClinGen/clincoded/wiki/VCI-Curation-Help page.
3. Click on the **Learn More** button towards the bottom of the Homepage and expect to be taken to a new window/tab showing either the https://github.com/ClinGen/clincoded/wiki/VCI-Curation-Help page.
4. Login and click on the **?** icons next to "Select Variant for Variant Curation" and "Create Gene-Disease Record" on the Dashboard.
5. Expect to be taken to a new window/tab showing either the https://github.com/ClinGen/clincoded/wiki/GCI-Curation-Help or https://github.com/ClinGen/clincoded/wiki/VCI-Curation-Help page.
6. Go to ^/create-gene-disease/ and click on the **Disease+** button and display to modal.
7. Click on the **Help** link in the modal and expect to be taken to a new window/tab showing either the https://github.com/ClinGen/clincoded/wiki/MONDO-Search-Help page.

**Steps to test (part 2):**
1. Signed in as a member of an affiliation and proceed with either creating a new GDM or edit an evidence with an existing GDM.
2. Return to the Dashboard page and expect to see the list items in the "Recent History" panel without the trailing "; last edited by XXX" text.
